README.md

```diff
@@ -1,1134 +1,233 @@
   - [Swagger Validations](#swagger-validations)
-  - [Test Generators](#test-generators)
-- [Building an Api-First-Hand Plugin](#building-an-api-first-hand-plugin)
+- [About API-First-Hand: Architecture and Structure](#about-api-first-hand-architecture-and-structure)
   - [Plugin Architecture](#plugin-architecture)
   - [Plugin Project Structure](#plugin-project-structure)
   - [Plugin Developing](#plugin-developing)
   - [Plugin Testing](#plugin-testing)
   - [Custom Templates For Code Generation](#custom-templates-for-code-generation)
-  - [Code Quality](#code-quality) 
+  - [Code Quality](#code-quality)
       
-## Compatibility
+## Intro to API-First-Hand
+**API-First-Hand** is an API-First bootstrapping tool for building RESTful web services from a [Swagger/OpenAPI](http://swagger.io/) specification. Constructed as a plugin, it takes your Swagger/OpenAPI definition as the single source of truth and regenerates these code snippets for you, simply and consistently. Instead of writing lots of boilerplate code, you can focus instead on implementing service business logic. Subsequent regenerations keep the code that you have added—either by commenting out the parts that are no longer valid, or by adding parts that are needed because you've changed the API.
 
-- Play 2.5.4+
-- Swagger (OpenAPI) 2.0
-
-## Build Status
+We built API-First-Hand for use with [Play Framework](http://www.playframework.com/), but we would like to extend it for use with **Akka HTTP**. [Get in touch](https://github.com/zalando/api-first-hand/blob/master/CONTRIBUTING.md) if you'd like to help make that possible.
 
-This plugin should be enabled using the [api-first-hand](http://www.typesafe.com/activator/template/api-first-hand) activator template 
-as the version in this repository is under active development. The status of this software is beta, 
-an end-to-end functional release intended to demonstrate the possibility to generate following from a Swagger specification:
+### Plugin Features 
 
-- Play route files
-- Generators of random test data
-- Wrappers for Play route files to convert semantics from http-related to domain-related (controller_base)
-- Skeletons for the domain-driven controller implementation
+Api-First-Hand supports round-trip regeneration and compilation of:
+- Play route definitions (managed)
+- Swagger domain model definitions and parameters onto Scala case classes (managed)
+- Swagger domain model constraints onto Play validations (managed)
+- Generators for random test data and parameter values (managed)
+- Unit tests for invalid and valid parameter sets // validating your service at the API boundary (managed)
+- Swagger path definitions onto skeletons for Play controller implementations (unmanaged)
+- Skeletons for domain-driven controller implementation and customized deserializers
+- Wrappers for Play route files to convert semantics from HTTP-related to domain-related (controller_base)
 - Model classes and validation rules
```

Comment:
Hey, sorry but I don't quite understand @slavaschmidt ... can you point me to where I duplicate?